docs/contribution.md

```diff
@@ -19,42 +19,39 @@ repository, it does not have to be lightweightm2m-iotagent, although we will use
 git remote add lightweightm2m-iotagent https://github.com/telefonicaid/lightweightm2m-iotagent.git
 ```
 
```

Comment:
More changes:

* L14: lightweightm2m-iotagent.git -> iotagent-json.git
* L17: lightweightm2m-iotagent -> iotagent-json
* L18: lightweightm2m-iotagent -> iotagent-json

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/220/commits/d1754b936268238b0b4b992c894cbc677d686e64